mmv1/products/identityplatform/Config.yaml

```diff
@@ -254,3 +254,116 @@ properties:
             description: |
               Two letter unicode region codes to allow as defined by https://cldr.unicode.org/ The full list of these region codes is here: https://github.com/unicode-cldr/cldr-localenames-full/blob/master/main/en/territories.json
             item_type: Api::Type::String
+  - !ruby/object:Api::Type::NestedObject
+    name: 'client'
+    description: |
+      Options related to how clients making requests on behalf of a project should be configured.
+    ignore_read: true
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'permissions'
+        description: |
+          Configuration related to restricting a user's ability to affect their account.
+        properties:
+          - !ruby/object:Api::Type::Boolean
+            name: 'disabledUserSignup'
+            description: |
+              When true, end users cannot sign up for a new account on the associated project through any of our API methods
+          - !ruby/object:Api::Type::Boolean
+            name: 'disabledUserDeletion'
+            description: |
+              When true, end users cannot delete their account on the associated project through any of our API methods
+      - !ruby/object:Api::Type::String
+        name: 'apiKey'
+        output: true
+        description: |
+          API key that can be used when making requests for this project.
+        sensitive: true
+        default_from_api: true
```

Comment:
I wonder what is the reason to add `default_from_api : true` here? 
`default_from_api : true` means that the key can be provided by the user. If the key is not provided by the user, it will use the returned one from API side. Is it true?